generator.yml

```diff
@@ -3,7 +3,6 @@ llvm_versions:
   - &llvm_latest 13
   - &llvm_12 12
   - &llvm_11 11
-  - &llvm_10 10
```

Comment:
I suspect we might want to keep some builds of stable with clang-10. We can remove this here, but let me file a bug about re-adding it for those.